src/screens/LoginScreen.tsx

```diff
@@ -94,6 +94,7 @@ export default function Login({ navigation }) {
         </TouchableOpacity>
       ) : undefined}
       {attemptInvisibleVerification && <FirebaseRecaptchaBanner />}
+      <Button title="BYPASS" onPress={() => navigation.navigate('Root')} />
```

Comment:
just set attemptInvisibleVerification to true, i think

https://docs.expo.dev/versions/latest/sdk/firebase-recaptcha/

also if you do this, do it as a separate PR please